ignition-sensors.rb

```diff
@@ -0,0 +1,155 @@
+class IgnitionSensors < Formula
+  desc "Common libraries for robotics applications. Sensors Library"
+  homepage "https://bitbucket.org/ignitionrobotics/ign-sensors"
+  url "https://bitbucket.org/ignitionrobotics/ign-sensors/get/3abcc7ff6b9b1e81b1acc1119a5038647864fe35.tar.gz"
```

Comment:
instead of adding patches, we can just point the URL to the tarball corresponding to the patch